lib/sidekiq/statistic/views/statistic.js

```diff
@@ -21,42 +21,11 @@ else{for(c=i&&i.getFullYear()===a,o=s&&s.getFullYear()===a,y+="<select class='ui
 $(function () {
   Chart.defaults.global.responsive = true
   Chart.defaults.global.scaleLineColor = 'rgba(0,0,0,.4)'
-  Chart.defaults.global.customTooltips = function(tooltip) {
```

Comment:
This apparently was not being used considering that `Charts` documentation declares custom tooltips different, inside tooltip options during initialization.